tests/acceptance/check-deleted-suites-in-expected-failure.sh

```diff
@@ -44,9 +44,17 @@ for expected_failure_file in "${EXPECTED_FAILURE_FILES[@]}"; do
   # Check the existence of the suite
   NONEXISTING_SCENARIOS=()
   for suite in "${EXPECTED_FAILURE_SUITES[@]}"; do
+    pattern="(\\b${suite}/[a-zA-Z0-9]+\\.feature:[0-9]+)"
     if [[ " ${AVAILABLE_SUITES[*]} " != *" $suite "* ]]; then
-      pattern="(${suite}/[a-zA-Z0-9]+\\.feature:[0-9]+)"
       NONEXISTING_SCENARIOS+=($(grep -Eo ${pattern} ${PATH_TO_EXPECTED_FAILURE_FILE}))
+    else
+      SCENARIOS=($(grep -Eo ${pattern} ${PATH_TO_EXPECTED_FAILURE_FILE} | grep -Eo "[a-zA-Z0-9]+\\.feature:[0-9]+"))
+      for scenario in "${SCENARIOS[@]}"; do
+        FEATURE_FILE=$(echo "$scenario" | cut -d':' -f1)
+        if [[ ! -f "$PATH_TO_SUITES/$suite/$FEATURE_FILE" ]]; then
+          NONEXISTING_SCENARIOS+=("$suite/$scenario")
```

Comment:
only for readability
```suggestion
          NON_EXISTING_SCENARIOS+=("$suite/$scenario")
```